### PR TITLE
Update README.md

### LINKED ISSUE #117: Update Directory path in README
Fix directory path in README at:
Building the soca-bundle
   5. Change the soca-config branch:  cd $CLONE_DIR/soca-bundle/soca-config
Add path to load python module

### PR DESCRIPTION
**Pull request by ShastriPaturi**
Description
0. Updates the README.md
    1. Loads the updated module files for python
    2. Updates the directory path to point to the correct soca-config branch
    3. The README.md has been updated.

Issue(s) addressed
Closes #117

Testing
Run the system.

### CODE REVIEW

**Review by flampouris — CHANGES_REQUESTED** [review]
Please update the original message of the PR to include description, issues addressed and how to test it, e.g. [link ](https://github.com/NOAA-EMC/godas/pull/114)

** Since PR #48 the detailed description of the PR is required **

File: README.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ Otherwise the RUNCDATE is created automatically at stmpX directory of the user.
 1. Load the JEDI modules \
    `module purge` \
    `source  $CLONE_DIR/modulefiles/godas.main` \
+   `source  $CLONE_DIR/modulefiles/godas.python` \
```
Comment (guillaumevernieres) [inline README.md:63]: Is this needed for the ioda-converters?

Comment (ShastriPaturi) [reply README.md:63]: Yes, it is needed.

**Comment by ShastriPaturi** [comment]
> Please update the original message of the PR to include description, issues addressed and how to test it, e.g. [link ](https://github.com/NOAA-EMC/godas/pull/114)
> 
> ** Since PR #48 the detailed description of the PR is required **

Updated the PR description

**Review by guillaumevernieres — APPROVED** [review]
All good.

**Review by flampouris — APPROVED** [review]
Nice!

**Pull request merged**